README.md

```diff
@@ -77,51 +79,55 @@ By contributing to this repository you agree to be a basic human being, please s
 
 ***TL;DR***: Initially during the early phase of development we'll keep this simple, after the library starts to become a dependency for real projects this will likely change.
```

Comment:
-"simple_ +"basic, -"real",  + "projects in real-world active use",  -"will likely change" +"the API will become considerably more extensive."